aibolit/patterns/return_null/return_null.py

```diff
@@ -20,31 +20,41 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-import javalang
-from collections import defaultdict
-
+from typing import List
 from aibolit.utils.ast_builder import build_ast
+from aibolit.utils.ast import AST, ASTNodeType
 
 
 class ReturnNull:
 
     def __init__(self):
         pass
 
-    def value(self, filename: str):
+    def get_return_args(self, node: int, tree: 'AST') -> bool:
+        for child1 in tree.children_with_type(node, ASTNodeType.LITERAL):
+            for child2 in tree.children_with_type(child1, ASTNodeType.STRING):
+                if tree.get_attr(child2, 'string') == 'null':
+                    return True
+        return False
+
+    def work_with_return(self, node: int, lines: List[int], tree: 'AST') -> List[int]:
```

Comment:
Same, you shouldn't use quotes on types here and should make method private.
This method do two things at once: some AST checks and appends line number to list. This two functionality should be separated.
Method name also does not gives any hint to its task.

Reply:
fixed